bower.json

```diff
@@ -20,6 +20,14 @@
     "isotope": "^3.0.4"
   },
   "overrides": {
+    "lazysizes": {
+      "main": [
+        "plugins/bgset/ls.bgset.min.js",
```

Comment:
does this order here matter? as you can see in the built files that these files are added to library.js in this order?